pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -111,8 +111,8 @@ public static AggregationFunction getAggregationFunction(FunctionContext functio
         } else if (numArguments == 2) {
           // Double arguments percentile (e.g. percentile(foo, 99), percentileTDigest(bar, 95), etc.) where the
           // second argument is a decimal number from 0.0 to 100.0.
-          // Have to use literal string because we need to cast int to double here.
-          double percentile = parsePercentileToDouble(arguments.get(1).getLiteral().getStringValue());
+          double percentile = arguments.get(1).getLiteral().getDoubleValue();
```

Comment:
i dont really think `.` can be added in function name `percentile99.5(col)` is not valid. (https://www.postgresql.org/docs/current/xfunc-sql.html)

Reply:
This is for `percentile(col, 99.5)` which is allowed